config/rbac/service_account.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: manager
+  namespace: system
+  
```

Comment:
there is now a blank line and it is missing the new line in the end

Reply:
@cpanato can you check now please?